pages/thank-you.jsx

```diff
@@ -19,6 +19,10 @@ var ThankYou = React.createClass({
   },
   renderBaseline: function() {
     var locale = this.props.locales[0];
+    var signUpOrSocial = (<Social language={locale}/>);
+    if (this.props.params && /^(?=.*\ben\b)|^(?=.*\bde\b)/.test(locale)) {
```

Comment:
Basically anything that starts with `en*` and `de` will get the signup the rest default to social.